### ISSUE TITLE
Show source and destination file sizes in completed queue?

### ISSUE THREAD

**Issue by wkearney99** [RandomEngy/VidCoder#959]
### Feature details

It'd be handy to easily see if the transcoding process led to larger/smaller file sizes when looking at the completed queue.  This to aid making the "Clear Succeeded" choice and deleting the source files.  There's relatively infrequent situations where a transcoding of a file actually making the size worse.  (This assumes 'quality' isn't necessarily the main goal)

An option to "customize columns" with a source file size would help for this.

**Comment by RandomEngy** [comment]
Those are good ideas.

**Comment by carlosalberto2000** [comment]
Yeah, I just migrate from Handbrake and Vidcoder has been amazing, but I miss this a lot (Source size, Target size, % difference).

**Comment by RandomEngy** [comment]
In v12.0 Beta, I've added the ability to customize columns in the Completed tab, and "Source size" and "% of source" columns.
